utils/config.py

```diff
@@ -3,6 +3,25 @@
 import os
 import json
 
+from core.config import load_bot_config_from_file, generate_bot_config
+
+try:
+    base_path = (Path(os.path.abspath(__file__)) / '..' / '..').resolve()
+    config_path = base_path / 'config_data.json'
+    config_data = load_bot_config_from_file(config_path.__str__()).get_store_json()
+except FileNotFoundError as fe:
+    print(fe)
+    print("BotConfig error: config_data.json file not found, generating new instance")
+    config_data = generate_bot_config({
```

Comment:
We could probably just use `BotConfig.factory` here to avoid the JSON roundabout.

Reply:
Either way works, whichever makes the most sense. I think the most intuitive method would be the one that most closely resembles the `config_data.json` file.